webapp/sources/rudder/rudder-rest/src/main/scala/com/normation/rudder/rest/lift/NodeApi.scala

```diff
@@ -636,12 +649,26 @@ class NodeApiService13 (
       n6              =  System.currentTimeMillis
       _               =  TimingDebugLoggerPure.logEffect.trace(s"all data fetched for response: ${n6 - n5}ms")
 
-      properties     <- req.json.flatMap(j => OptionnalJson.extractJsonListString(j, "properties").map(_.getOrElse(Nil)))
-      mapProps       <- getNodesPropertiesTree(nodes.values, properties).toBox
+      properties     <- req.json.flatMap(j => OptionnalJson.extractJsonArray(j, "properties")(json => extractNodePropertyInfo(json)  )).map(_.getOrElse(Nil))
+
+      mapProps       <- properties.partition(_.inherited) match {
+        case (inheritedProp, nonInheritedProp) =>
+          val propMap = nodes.values.groupMapReduce(_.id)(n =>  n.properties.filter(p => nonInheritedProp.exists(_.value.split("\\.").headOption.map(_ == p.name).getOrElse(false))))(_ ::: _)
```

Comment:
you should normalize property name set before giving it to methods below. It would be hard to maintain that code it people need to know that properties are not the correct name, but a specific interchange format with datatable

Reply:
That because a i allow access to sub element of a property